### PR TITLE
(WIP) update readme.md

### PR DESCRIPTION
**Pull request by maltenuhn**
**Problem:**
Our readme was a bit hard to follow, and required you to read it sequentially. 

**Fix:**
I've tried to make it more readable and focused the top section on the casual first-time installer - separating the "make your life easier" parts from "just get something running". I _think_ it's a better direction, but wanted to get your thoughts on it before continuing.

### CODE REVIEW

**Comment by balazsbajorics** [comment]
looks good to me. the prerequisites are more readable with the reduced clutter.

File: readme.md [hunk 1]
```diff
@@ -11,47 +11,70 @@ Utopia is an integrated design and development environment for React. It uses Re
...
+- (MacOS only: Install [brew](https://brew.sh/) and run run `brew install gmp`. On an M1 Mac, follow [these steps](https://github.com/concrete-utopia/utopia#m1-macbook)
+- Install [nix-shell](https://nixos.org/download.html). On Windows? Follow [this guide](https://nathan.gs/2019/04/12/nix-on-windows/). 
+- Recommended: Install [direnv](https://github.com/concrete-utopia/utopia#using-direnv-to-make-your-life-easier). 
+
+### Run the Editor for the first time
+
+Install the prerequsites, then run this in your terminal. NB There's no docker image at the moment.
 
 ```
+nix-shell
```
Comment (seanparsons) [inline readme.md:43]: You can make this `nix-shell --run start-full` and it'll run that within the shell once the dependencies have been pulled.

**Review by enidemi — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]
